pom.xml

```diff
@@ -29,28 +29,53 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
-
         <dependency>
             <groupId>com.h2database</groupId>
             <artifactId>h2</artifactId>
             <scope>runtime</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
just move the things exactly back where they were line to line, and it should work

Reply:
There is also some sort of an error connecting to the liquibase, which I am not sure we need for this task, do we? so let's remove it